### ISSUE TITLE
Generic error when the jar is missing

### ISSUE THREAD

**Issue by nevir** [vvo/selenium-standalone#57]
When running `selenium.start` without having the jar in place, you end up with a generic error (timeout). Highlights two things, actually:

* IMO there should be a specific error for this case (it'll probably be common for API usage)
* `selenium.start` should listen to the `exit` event and generate an error if it has yet to start

**Comment by vvo** [comment]
> IMO there should be a specific error for this case (it'll probably be common for API usage)

Yes, should be fairly easy to check, a check on the existence of [paths.selenium](https://github.com/vvo/selenium-standalone/blob/f2bb644e02cafe16fc3ee7ffd8af9cdc7849a8b1/lib/start.js#L37) should be enough

Will accept PR for this.

> selenium.start should listen to the exit event and generate an error if it has yet to start

Sorry I do not understand this request, can you rephrase it

**Comment by vvo** [comment]
> selenium.start should listen to the exit event and generate an error if it has yet to start

Ok so now I understand th exit thing, let me handle all this for you.

**Comment by nevir** [comment]
Awesome, thanks!

**Comment by nevir** [comment]
extra clarification, just in case: The child process dies immediately if the jar is missing (it emits `exit`), but selenium-standalone misses that event, so it keeps waiting until it times out
